### PR TITLE
Docs on exclude_data_from_backup parameter

### PR DESCRIPTION
**Pull request by kathancox**
Fixes [DOC-3286](https://cockroachlabs.atlassian.net/browse/DOC-3286)

Added docs on new `exclude_data_from_backup` parameter. Included:

- Longer examples on Take Full and Incremental Backups page
- New page for the `SET (storage parameter)` subcommand of `ALTER TABLE`
- Short-form example on `CREATE TABLE`
- Notes/considerations to `BACKUP` & `RESTORE` pages

### CODE REVIEW

**Review by livlobo — APPROVED** [review]
LGTM

File: v22.1/take-full-and-incremental-backups.md [hunk 1]
```diff
@@ -311,10 +485,95 @@ Both core and Enterprise users can use backup scheduling for full backups of clu
...
 
+### Exclude a table's data from backups
+
+<span class="version-tag">New in v22.1:</span> In some situations, you may want to exclude a table's row data from a [backup](backup.html). For example, you have a table that contains high-churn data that you would like to [garbage collect](architecture/storage-layer.html#garbage-collection) more quickly than the [incremental backup](#incremental-backups) schedule for the database or cluster holding the table. You can use the `exclude_data_from_backup = true` parameter with a [`CREATE TABLE`](create-table.html#create-a-table-with-data-excluded-from-backup) or [`ALTER TABLE`](set-storage-parameter.html#exclude-a-tables-data-from-backups) statement to mark a table's row data for exclusion from a backup.
+
+It is important to note that the backup will still contain the table, but it will be empty. Also, setting this parameter prevents the cluster or database backup from delaying [GC TTL](configure-replication-zones.html#gc-ttlseconds) on the key span for this table. This is useful when you want to set a shorter garbage collection window for tables containing high-churn data to avoid an accumulation of unnecessary data.
```
Comment (adityamaru) [inline v22.1/take-full-and-incremental-backups.md:494]: nit: prevents the cluster or database backup from delaying GC on the key span for this table, and respects the configured GC TTL.

**Review by adityamaru — APPROVED** [review]

**Comment by kathancox** [comment]
Hey @nickvigilante it would be great to get your review & feedback on this since some of your pages/product area are included here.

**Review by nickvigilante — APPROVED** [review]
:lgtm: Great work! I think this will help set up a nice framework for me to document other storage parameters.

Small nit that's entirely outside the scope of this ticket, but I think we're repeating a lot of code and text with the current section layout. Essentially, your example doesn't change from section to section, so after this gets published, I'm gonna test to see if I can lean out some of the code. But that's not required at all for this ticket.

> Reviewed 5 of 8 files at r1, 1 of 1 files at r2, 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13773#-:-N2DBrchELpme1vr1svb:b-x3qq6n)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @kathancox)

**Pull request merged**